common/src/main/java/org/opensearch/ml/common/MLTaskType.java

```diff
@@ -9,5 +9,7 @@ public enum MLTaskType {
     TRAINING,
     PREDICTION,
     TRAINING_AND_PREDICTION,
-    EXECUTION
+    EXECUTION,
+    UPLOAD_MODEL,
+    LOAD_MODEL
```

Comment:
What about UNLOAD_MODEL? are you planning to add later?

Reply:
Actually unload model part is quite lightweight (just remove some objects from memory), I think we don't need to create ML task for it. Creating a new unload task may add too much overhead comparing the simple unload action.